spec/spec_helper.rb

```diff
@@ -22,21 +23,52 @@ def establish_database_connection
   ActiveRecord::Base.establish_connection(
     adapter: "postgresql",
     host: ENV.fetch("PGHOST", "localhost"),
-    user: ENV.fetch("PGUSER", "postgres"),
-    password: ENV.fetch("PGPASSWORD", ""),
+    user: ENV.fetch("PGUSER", "ubuntu"),
+    password: ENV.fetch("PGPASSWORD", "password"),
     database: ENV.fetch("PGDATABASE", "que-test"),
   )
 end
 
 establish_database_connection
 
+
+class LockDatabaseRecord < ActiveRecord::Base
+  establish_connection(
+      adapter: "postgresql",
+      host: ENV.fetch("LOCK_PGHOST", "localhost"),
+      user: ENV.fetch("LOCK_PGUSER", "ubuntu"),
+      password: ENV.fetch("LOCK_PGPASSWORD", "password"),
+      database: ENV.fetch("LOCK_PGDATABASE", "lock-test"),
+      port: ENV.fetch("LOCK_PGPORT", 5435),
+      pool: 6,)
+end
+
+class YugabyteRecord < ActiveRecord::Base
```

Comment:
Let's keep the Que repo database agnostic, it just needs to work with Postgres
```suggestion
class JobRecord < ActiveRecord::Base
```